packages/contracts/src/networks.ts

```diff
@@ -63,8 +63,8 @@ export const SPHINX_LOCAL_NETWORKS: Array<SupportedLocalNetwork> = [
 ]
 
 export type NetworkType = 'Testnet' | 'Mainnet' | 'Local'
-type RollupProvider = 'Conduit' | 'Caldera'
-type RollupType = 'OP Stack' | 'Arbitrum'
+type RollupProvider = 'Conduit' | 'Caldera' | 'None'
```

Comment:
nit: the `null` type may be preferable instead of a string called "None"

Reply:
When we eventually move all of this information into the DB, it'll be better to have a defined value vs null. That provides better type safety when interacting with the DB.